example/src/main/tut/typeclass/Show.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: docs
+title:  "Show"
+---
+
+# Show
+
+`Show` presents a safe and explicit alternative to `Object.toString`.
+By providing instances of this typeclass, a type explicitly defines that it can be converted into human-readable output.
+
+**Typical imports**
+
+```tut:silent
+import scalaz.Prelude._
+```
+
+# Built-in instances
+
+```tut
+val stringShow: Show[String] = implicitly
```

Comment:
`Show` has a companion `apply` that looks better IMO

```scala
val stringShow: Show[String] = Show[String]
```

Reply:
This was just meant to show that instances for standard types are already (implicitly) available